### PR TITLE
[Celo sage -- John] 5-11-2023-how-to-develop-your-game-items-with-thirdweb-erc1155-base-contract-on-celo

### PR DESCRIPTION
**Pull request by johnfawole**

### CODE REVIEW

File: blog/5-11-2022-how-to-develop-your-game-items-with-thirdweb-erc1155-base-contract-on-celo/thirdweb.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+Go to the extension section, search for sourcify, and activate it. Once activated, click on it on the left corner of the screen, click on verifier, and provide the network. Since we deployed on Celo Alfajores testnet, we'll select that. Provide the contract address, select the appropriate contract name, click on verify, and your contract will be verified in seconds.
+
+![thirdweb3](https://github.com/celo-org/docs/assets/105144630/c5fc6c19-4fb4-4923-a89e-2aef3fa204f2)
+![thirdweb4](https://github.com/celo-org/docs/assets/105144630/e87ea392-38d2-4d95-9bf7-821d5df639f5)
+![thirdweb5](https://github.com/celo-org/docs/assets/105144630/4a437ac4-4d20-4762-af8f-b17f1540f5b7)
+
+You can now visit the block explorer to inspect and interact with your contract.
+
+## Concluding Remarks
```
Comment (yafiabiyyu) [inline blog/5-11-2022-how-to-develop-your-game-items-with-thirdweb-erc1155-base-contract-on-celo/thirdweb.md:264]:
```suggestion
## Conclusion
```

**Review by yafiabiyyu — COMMENTED** [review]
## Review Result

1. The `tags` in the tutorial data are markdown and also in the `users.tsx` need to be fixed.
2. Code formatting needs to be fixed to make it neater.

File: blog/5-11-2022-how-to-develop-your-game-items-with-thirdweb-erc1155-base-contract-on-celo/thirdweb.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        return baseContract.getRoyaltyInfoForToken(tokenId);
+    }
+
+    function calculateRoyalty (uint tokenId, uint salePrice) external view returns (uint) {
+        ( , uint royaltyValue) = baseContract.royaltyInfo(tokenId, salePrice);
+        return royaltyValue;
+    }
+```
+
+We have 3 functions for royalties. setRoyaltyInfo sets the royalty receiver and the royalty rate for a specific collection. getRoyaltyInfo gets the receiver address and royalty rate for a particular collection. Finally, calculateRoyalty get the royalty due on an NFT in a collection given a particular sale price.
```
Comment (yafiabiyyu) [inline blog/5-11-2022-how-to-develop-your-game-items-with-thirdweb-erc1155-base-contract-on-celo/thirdweb.md:208]:
```suggestion
We have 3 functions for royalties. `setRoyaltyInfo` sets the royalty receiver and the royalty rate for a specific collection. `getRoyaltyInfo` gets the receiver address and royalty rate for a particular collection. Finally, `calculateRoyalty` get the royalty due on an NFT in a collection given a particular sale price.
```

**Review by yafiabiyyu — COMMENTED** [review]
There are several revisions that allow the use of block codes in writing. However, there is one function that does not explain in detail the parameters needed for that function.

**Comment by viral-sangani** [comment]
Closing Celo Sage PRs

**Pull request closed without merging**